chronograf/server/config.go

```diff
@@ -12,6 +12,10 @@ type configLinks struct {
 	Auth string `json:"auth"` // Auth link to the auth config endpoint
 }
 
+type selfLinks struct {
```

Comment:
What's this for?

Reply:
This was the only code in `sources.go` that seemed needed so I moved it out. I mean to comment and make sure to bring it to attention during review.